CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ _The old changelog can be found in the `release-2.6` branch_
 
   - Add http/https protocols for singularity run/pull commands
   - Update to SIF 1.0.2
+  - Add _noPrompt_ parameter to `pkg/signing/Verify` function to enable silent verification
```

Comment:
I would leave this out of the changelog personally. I think we want to reserve changelog for things that change how someone uses Singularity. For example, a change to CLI flags or a behavior change in a command.

Reply:
Even if it is a public package?

Reply:
I see your point. Not too big of a deal IMO so we can just leave it